uber/models.py

```diff
@@ -1167,7 +1165,7 @@ def is_dept_head(self):
 
     @property
     def can_check_in(self):
-        return self.paid != c.NOT_PAID and self.badge_status == c.COMPLETED_STATUS and not self.is_unassigned
+        return self.paid != c.NOT_PAID and self.badge_status in [c.NEW_STATUS, c.COMPLETED_STATUS] and not self.is_unassigned
```

Comment:
There's at least one situation where we need to be able to check in people whose badge status is ``c.NEW_STATUS``: people claiming unassigned group badges.  Their registrations are correctly ``c.NEW_STATUS``, because they're marked as ``c.PAID_BY_GROUP`` without actually having a group.  Therefore, we need the button to show up for people with ``c.NEW_STATUS`` badges, and then have the backend block the check-in action if the requirements aren't met when the button is pressed.